From af1e2c9506249e0f619fcb3258c7d276e9473a48 Mon Sep 17 00:00:00 2001
From: Sandeep Kumar <sandeep4.kumar@intel.com>
Date: Mon, 8 Sep 2025 15:29:06 +0530
Subject: [PATCH 2/2] enabling the custom syscall

---
 arch/x86/entry/syscalls/syscall_64.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/entry/syscalls/syscall_64.tbl b/arch/x86/entry/syscalls/syscall_64.tbl
index c84d12608..af4f1d69e 100644
--- a/arch/x86/entry/syscalls/syscall_64.tbl
+++ b/arch/x86/entry/syscalls/syscall_64.tbl
@@ -372,6 +372,7 @@
 448	common	process_mrelease	sys_process_mrelease
 449	common	futex_waitv		sys_futex_waitv
 450	common	set_mempolicy_home_node	sys_set_mempolicy_home_node
+452 common  migrate_dst_tier        sys_migrate_dst_tier
 
 #
 # Due to a historical design error, certain syscalls are numbered differently
-- 
2.43.0

